pkg/plugin/v2/plugin.go

```diff
@@ -85,10 +103,13 @@ func (m *Manager) Start(
 		}
 	}
 
+	port := 13370
 	// after loading all plugins we start them
 	for _, vClusterPlugin := range m.Plugins {
 		// build the start request
-		initRequest, err := m.buildInitRequest(filepath.Dir(vClusterPlugin.Path), syncerConfig, vConfig)
+		initRequest, err := m.buildInitRequest(filepath.Dir(vClusterPlugin.Path), syncerConfig, vConfig, port)
+		port++
```

Comment:
Is it possible to check if the port is available?

Reply:
how would it react if the port isn't available? should we exit ? If so that's kinda equivalent to the current code. I am not sure if changing the port if there's already something listening on it is a good idea. That should be a reserved range for vcluster plugins and so failing is ok if something is listening on it

Reply:
> how would it react if the port isn't available? should we exit ?

We might exit, we might also omit any port that's in use, and try the next one.

> That should be a reserved range for vcluster plugins and so failing is ok if something is listening on it

Is this reserved port range documented somewhere?

Reply:
not yet, but also even if we check that the port is not in use now, there's no guarantee that the port will be available when the plugin starts, so it's not really changing anything

Reply:
this can be done from the sdk though, and that will be reliable